Solution/iOS/RocketAppUITests/Screens/LoginScreen.swift

```diff
@@ -7,15 +7,41 @@ struct LoginScreen: Screen {
   let app: XCUIApplication
 
   private let loginTitle: XCUIElement
+  private let usernameField: XCUIElement
+  private let passwordField: XCUIElement
+  private let loginButton: XCUIElement
 
   init(app: XCUIApplication) {
     self.app = app
     loginTitle = app.staticTexts[AccessibilityKeys.Login.title]
+    usernameField = app.textFields[AccessibilityKeys.Login.username]
+    passwordField = app.secureTextFields[AccessibilityKeys.Login.password]
+    loginButton = app.buttons[AccessibilityKeys.Login.loginButton]
   }
 
   @discardableResult
   func checkLoginTitle() -> Self {
-    XCTAssert(loginTitle.waitForExistence(timeout: 5))
+    XCTAssert(loginTitle.waitForExistence(timeout: 5), "Login title did not appear within the timeout.")
+    return self
+  }
+  
+  @discardableResult
+  func enterUsername(_ username: String) -> Self {
```

Comment:
enter(username: String) to iste potom dal, Swift preferuje tuto syntax, mrkni na best practices